src/components/discover-sheet/DiscoverSheetContent.js

```diff
@@ -24,7 +24,7 @@ function Switcher({ showSearch, children }) {
   return (
     <>
       <View style={{ display: showSearch ? 'flex' : 'none' }}>
-        {children[0]}
+        {showSearch ? children[0] : <FlexItem />}
```

Comment:
so children[0] was returning different values all the time?

Reply:
yes, aside from changing every time an asset changed, it was always rendered with display: none